apps/src/templates/lessonOverview/ResourceList.jsx

```diff
@@ -16,11 +23,58 @@ export default class ResourceList extends Component {
     }
   };
 
+  downloadResource = resource => {
+    firehoseClient.putRecord(
+      {
+        study:
+          this.props.pageType === 'resources-rollup'
+            ? 'rollup-pages'
```

Comment:
what do you think about `course-rollup-pages`? otherwise it sounds a bit general in a sea of study types